### ISSUE TITLE
[Bug]: grafana_oncall_outgoing_webhook: new trigger_type = status change is not implemented

### ISSUE THREAD

**Issue by xe-leon** [grafana/terraform-provider-grafana#1873]
### Terraform Version

1.9.6

### Terraform Grafana Provider Version

3.10.0

### Grafana Version

Grafana Cloud (steady)

### Affected Resource(s)

* grafana_oncall_outgoing_webhook

### Terraform Configuration Files

```
resource "grafana_oncall_outgoing_webhook" "changed" {
  name                  = "[tf] changed-test"
  is_webhook_enabled    = true
  trigger_type          = "status change"
  forward_whole_payload = false
  http_method           = "PUT"
  trigger_template      = "1"
  url                   = "https://example.com/"
  data                  = "any data"
}

terraform {
  required_providers {
    grafana = {
      source  = "grafana/grafana"
    }
  }
}

provider "grafana" {
  oncall_url          = "https://oncall-prod-eu-west-0.grafana.net/oncall"
  oncall_access_token = "<oncall-api-token>"
  retry_wait          = 60
  retries             = 15
}
```

### Expected Behavior

outgoing webhook created in grafana, resource saved in terraform state

### Actual Behavior

outgoing webhook created in grafana, but resource is NOT saved in terraform state since provider fails with the error:
```
Error: POST https://oncall-prod-eu-west-0.grafana.net/oncall/api/v1/webhooks/: 400 {non_field_errors: [The fields name, organization must make a unique set.]}
with grafana_oncall_outgoing_webhook.changed,
on main.tf line 3, in resource "grafana_oncall_outgoing_webhook" "changed":
3: resource "grafana_oncall_outgoing_webhook" "changed" {
```

### Steps to Reproduce

1. terraform init
2. terraform apply
3. check grafana cloud UI. Webhook has been created
4. provider fails, thus not saving resource to terraform state

### References

- https://github.com/grafana/oncall/pull/3920
- [provider documentation](https://registry.terraform.io/providers/grafana/grafana/latest/docs/resources/oncall_outgoing_webhook#trigger_type). There's no such option as a `status change`

**Comment by prein** [comment]
The error you are observing looks like it has to do with uniqueness of the `name` value. Did you figure it out in the meantime?

**Comment by xe-leon** [comment]
I don't think it has something to do with `name` value, because if I manually create this webhook and then import it using `import` block in terraform, I can modify it using terraform. The only issue here is you cannot create outgoing webhook with this trigger type.
